frontend/mina-signer/src/TSTypes.ts

```diff
@@ -0,0 +1,46 @@
+export type uint32 = number | bigint | string;
+export type uint64 = number | bigint | string;
+
+export declare type publicKey = string;
```

Comment:
Why is `export declare type ...` used throughout this file instead of `export type ...`? IIUC declare has to be used on stuff that represents a *value* (for which you declare a type), on types it doesn't seem to do anything?

Reply:
Fixed!